### DISCUSSION TITLE
500 error to sign in or sign up into supabase

### DISCUSSION THREAD

**Discussion by milestones95** [supabase/supabase#2571] [Questions] [answered]
Hi I'm not sure why it's not letting me sign in or sign up on supabase. Is anyone else hitting this issue?

Request URL: https://app.supabase.io/api/profile

Error message: {"message":"invalid json response body at https://readonly.supabase.io/rest/v1/users?select=id%2Cauth0_id%2Cprimary_email%2Cusername%2Cfirst_name%2Clast_name%2Cmobile%2Cis_alpha_user%2Corganizations%28id%2Cslug%2Cname%2Cbadge%2Cproject_limit%2Cstripe_customer_id%2Cbilling_email%2Cstripe_customer_object%2Cprojects%28id%2Cref%2Cname%2Cstatus%2Corganization_id%2Cinserted_at%2Csubscription_id%2Cdb_user_supabase%2Cdb_host%2Cdb_pass_supabase%2Cdb_ipv4%2Cdb_port%2Cdb_name%2Cdb_ssl%2Cinfrastructure_config-%3E%22cloud_provider%22%2Cinfrastructure_config-%3E%22region%22%2Cservices%28infrastructure%28app_versions%28version%29%29%29%29%29&auth0_id=eq.github%7C47959617&organizations.status=neq.REMOVED&organizations.projects.status=neq.GOING_DOWN&organizations.projects.status=neq.REMOVED&organizations.projects.services.status=neq.REMOVED&organizations.projects.services.infrastructure.status=neq.REMOVED reason: Unexpected token A in JSON at position 0","type":"invalid-json"}

**Comment by 72L** [comment, marked as answer]
Yes, there's an issue filed here: https://github.com/supabase/supabase/issues/2568

~~For immediate relief, you can try using a VPN. Singapore works for me!~~ (no longer)

**Comment by CaseyG5** [comment]
Beginning yesterday and still today `supabase.auth.signUp()` is not working.  I've also tried HTTP requests without the `SupabaseClient` (both with Axios in my application and via HTTP client in my IDE) but still I'm getting error 500.  Is this part of Supabase down atm?

`{`
`  "code": 500,`
`  "msg": "Database error saving new user",`
`  "error_id": "e3a10df5-fb34-40f7-8f27-8aa5457805b3"`
`}`

`Response code: 500 (Internal Server Error); Time: 217ms; Content length: 101 bytes`

Thanks in advance if you have any info/update on this.
